packages/slice-machine/.storybook/main.ts

```diff
@@ -0,0 +1,10 @@
+import type { StorybookConfig } from "@storybook/react-vite";
+
+const config: StorybookConfig = {
+  addons: ["@storybook/addon-essentials"],
+  docs: { autodocs: "tag" },
+  framework: "@storybook/react-vite",
+  stories: ["../src/**/*.stories.ts?(x)"],
```

Comment:
❓ Will we have a story .ts and not .tsx? I say that because I don't think I created a one without .tsx in the past

Reply:
I mean not really critical, it was more a question like I don't know why we would use that. Not a problem to keep it if it's the default. As you want ;)